bazel/scripts/check_py_bazel.sh

```diff
@@ -0,0 +1,135 @@
+    "./lte/gateway/deploy"
+    "./lte/gateway/python/magma/tests/pylint_wrapper.py"
+    "./lte/gateway/python/precommit.py"
+    "./orc8r/cloud/deploy"
+    "./orc8r/cloud/docker"
+    "./orc8r/tools"
+    "./protos"
+    "./show-tech"
+    "./third_party"
+    "./xwf/gateway/deploy"
+)
+
+# folders and files that are relevant for building with bazel
+# this list needs to be updated if respected structures are bazelified
+DENY_LIST_NOT_YET_BAZELIFIED=(
+    "./dp/cloud/python/magma"
+    "./lte/gateway/c/core/oai/tasks/s1ap/messages/asn1"
+    "./lte/gateway/python/integ_tests"
+    "./lte/gateway/python/load_tests"
+    "./lte/gateway/python/scripts"
+    "./orc8r/gateway/python/scripts"
+    "./orc8r/gateway/python/magma/magmad/tests/dummy_service.py"
+    "./orc8r/gateway/python/magma/magmad/upgrade/docker_upgrader.py"
+    "./orc8r/gateway/python/magma/common/health/docker_health_service.py"
+    "./orc8r/gateway/python/magma/common/health/health_service.py"
+    "./orc8r/gateway/python/magma/common/health/entities.py"
+    "./lte/gateway/python/magma/health/health_service.py"
+    "./lte/gateway/python/magma/health/entities.py"
+    "./lte/gateway/python/magma/mobilityd/tests/rpc_servicer_tests.py"
+    "./lte/gateway/python/magma/pkt_tester/tests/test_topology_builder.py"
+    "./lte/gateway/python/magma/pkt_tester/tests/test_ovs_gtp.py"
+    "./lte/gateway/python/magma/pkt_tester/topology_builder.py"
+    "./lte/gateway/python/magma/pipelined/tests/script/gtp-packet.py"
+    "./lte/gateway/python/magma/pipelined/tests/script/ip-packet.py"
+    "./lte/gateway/python/magma/pipelined/tests/envoy-tests/http-serve.py"
+    "./lte/gateway/python/magma/pipelined/openflow/events.py"
+    "./lte/gateway/python/magma/pipelined/pg_set_session_msg.py"
+    "./lte/gateway/python/magma/enodebd/tr069/tests/models_tests.py"
+    "./lte/gateway/python/magma/enodebd/state_machines/enb_acs_pointer.py"
+)
+
+DENY_LIST=( "${DENY_LIST_NOT_RELEVANT[@]}" "${DENY_LIST_NOT_YET_BAZELIFIED[@]}" )
+
+# files that exists in multiple folders but ar generally not covered by bazel
+NOT_RELEVANT_FILES=(
+  "__init__.py"
+  "setup.py"
+  "pylint_tests.py"
+  "fabfile.py"
+)
+
+DENY=()
+FIRST_ITERATION=true
+for entry in "${DENY_LIST[@]}"
+do
+  if [ "$FIRST_ITERATION" = true ]
+  then
+    DENY+=( "-path" "$entry" )
+    FIRST_ITERATION=false
+  else
+    DENY+=( "-o" "-path" "$entry" )
+  fi
+done
+
+ALL_PY_FILES=$(find . \( "${DENY[@]}" \) -prune -o -iname "*.py" -print)
+
+echo "Searching files that are not covered by a BUILD.bazel file ..."
+
+NOT_IN_BUILD_FILE=()
+for file in $ALL_PY_FILES
+do
+	PY_PATH=$(dirname "$file")
+	PY_FILE=$(basename "$file")
+
+  if [[ " ${NOT_RELEVANT_FILES[*]} " =~ ${PY_FILE} ]]
+  then
+		continue
+	fi
+
+	BUILD_FILE="${PY_PATH}/BUILD.bazel"
+	if test -f "$BUILD_FILE"
+  then
+		if ! grep -q "$PY_FILE" "$BUILD_FILE";
+		then
+      NOT_IN_BUILD_FILE+=( "$file" )
+		fi
+	else
+    	NOT_IN_BUILD_FILE+=( "$file" )
+	fi
```

Comment:
The indentation seems to be mixed up here.

Reply:
fixed with refactoring